sdk/storage/storage-file-share/review/storage-file-share.api.md

```diff
@@ -1705,6 +1700,11 @@ export interface ShareProperties {
     remainingRetentionDays?: number;
 }
 
+// @public
+export interface ShareProtocolSettings {
```

Comment:
Has this been release before? since renaming would be a breaking change?

Reply:
New feature. Hasn't been released before.